tpgtools/overrides/compute/beta/network_firewall_policy_rule.yaml

```diff
@@ -0,0 +1,19 @@
+- type: CUSTOM_RESOURCE_NAME
+  details:
+    title: region_network_firewall_policy_rule
+  location: region
+- type: EXCLUDE
+  field: location
+  location: global
+- type: EXCLUDE
+  field: region
```

Comment:
non-blocking: I'm guessing this one's because we're auto-injecting a field called `region` based on the ID, right?

Reply:
I cheated from the override file for [forwarding rule](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/tpgtools/overrides/compute/forwarding_rule.yaml) for this one. But I believe so yes.